test/test_kubeclient.rb

```diff
@@ -38,6 +38,18 @@ def test_no_version_passed
     assert_equal 'http://localhost:8080/api/v1', rest_client.url.to_s
   end
 
+  def test_pass_proxy
+    uri = URI::HTTP.build(host: 'localhost', port: 8080)
+    proxy_uri = URI::HTTP.build(host: 'myproxyhost', port: 8888)
+    client = Kubeclient::Client.new(uri, http_proxy_uri: proxy_uri.to_s)
+    rest_client = client.rest_client
+    assert_equal proxy_uri.to_s, rest_client.options[:proxy]
+
+    watch_client = client.watch_pods
+    assert_equal(watch_client.send(:build_client_options)[:proxy][:proxy_address], proxy_uri.host)
```

Comment:
please use the same style on lines 48 and 49 - either assert_equal with brackets, or without.